content/applications/hr/payroll/payslips.rst

```diff
@@ -221,166 +304,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-   :align: center
-   :alt: Generate payslips from the edited batch.
+.. _payroll/batch-process:
 
-Once the draft payslips are created, the button changes to say :guilabel:`Make Payment`. Click the
-:guilabel:`Make Payment` button. A pop-up window appears, in which the bank journal information must
-be entered. Select the :guilabel:`Bank Journal` from the drop-down list, and click
-:guilabel:`Confirm` to process the payslips, and pay the employees.
+Process a batch
+---------------
 
-On the batch detail page, the number of payslips in the batch is accessible via the
-:guilabel:`Payslips` smart button, located in the top-right corner. Individual payslips for the
-batch can be viewed by clicking the :guilabel:`Payslips` smart button in the top-right corner.
+Click on an individual batch to view the details for that batch on a separate page. On this batch
+detail page, different options (buttons) appear at the top, depending on the status of the batch:
 
-Use the breadcrumb menu to navigate back to the individual batch detail page, or back to the list of
-all batches.
+- :guilabel:`New` status: batches without any payslips added to them have a status of
+  :guilabel:`New`. The following button options appear for these batches:
 
-.. image:: payslips/payslip-batches.png
-   :align: center
-   :alt: Click the Payslips smart button to view the individual payslips in the batch.
+  .. image:: payslips/batch-new.png
+     :align: center
+     :alt: A batch with a status of new, with the available buttons highlighted.
 
-Create a new batch
-------------------
+  - :guilabel:`Add Payslips`: click the :guilabel:`Add Payslips` button to add payslips to the
+    batch, and an :guilabel:`Add Payslips` pop-up window appears. Only payslips that can be added to
+    the batch (payslips not currently part of a batch) appear on the list.
 
-To create a new batch of payslips from the :guilabel:`Payslips Batches` page
-(:menuselection:`Payroll app --> Payslips --> Batches`), click the :guilabel:`Create` button in the
-upper-left corner. Doing so reveals a blank payslip batch form on a separate page.
+    Select the desired payslips by clicking the checkbox to the left of each payslip name, then
+    click the :guilabel:`Select` button to add them to the batch. Once payslips are selected and
+    added to the batch, the status changes to :guilabel:`Confirmed`.
 
-On the new payslip batch form, enter the :guilabel:`Name` for the batch, and select the date range
-to which the batch applies, by clicking the :guilabel:`▼ (drop-down arrow)` icon in the
-:guilabel:`Period` fields, which reveals a calendar pop-up window. From this calendar pop-up window,
-navigate to the correct month, and click on the corresponding day for both the start and end dates.
+   - :guilabel:`Generate Payslips`: after payslips have been added to the batch, click the
+     :guilabel:`Generate Payslips` button to process the payslips and create individual payslips in
+     the database.
```

Comment:
If this bullet (:guilabel:\`Generate Payslips\`:) is intended to be a sub-bullet to (:guilabel:\`Add Payslips\`:), please indent it by another space, otherwise please scoot it back a space to align with its parent bullet point
```suggestion
    - :guilabel:`Generate Payslips`: after payslips have been added to the batch, click the
      :guilabel:`Generate Payslips` button to process the payslips and create individual payslips in
      the database.
```